Cargo.toml

```diff
@@ -17,11 +17,14 @@ codegen-units = 1
 panic = "abort"
 debug = true
 
+[patch."https://github.com/gakonst/ethers-rs"]
+ethers = "0.6.2"
```

Comment:
Nix builds run without network access for reproduceability. naersk (rust flake helper I'm using) fails to detect sub-workspaces' externally-accessed dependencies, so overriding here fixes this particular problem (but it's just a crutch for now).